CONTRIBUTING.md

```diff
@@ -6,7 +6,10 @@ There are many ways in which you can contribute, beyond writing code. The goal o
 
 ## How to contribute
 
-1. Fork this very repository (use the `fork` button at the top left)
+1. Fork this very repository (use the `fork` button at the top left).
+
+**Tip**: If you have permission, use the [GeeksCAT fork](https://github.com/GeeksCAT/anem-per-feina) directly instead of create a new fork.
```

Comment:
This **Tip** here is understandable?

Reply:
I think that it should be something like: `**Tip for hacktoberfest'20 participants**: If you have doubts or troubles forking and keeping it synchronized, start a new `branch` from `master` and submit a PR directly in this repo.`

Reply:
Great! It's more explicit, and explicit is better than implicit :)